drivers/wifi/nrf700x/osal/fw_if/umac_if/inc/fmac_api.h

```diff
@@ -201,9 +199,8 @@ enum wifi_nrf_status nrf_wifi_fmac_set_xo_val(struct wifi_nrf_fmac_dev_ctx *fmac
 						 unsigned char value);
 
 /**
- * nrf_wifi_fmac_rf_test_get_xo_value() - Get XO calibrated value.
+ * nrf_wifi_fmac_rf_test_compute_optimal_xo_val() - Get XO calibrated value.
```

Comment:
Please use short forms where possible so that code can fit in the line limit of 100 chars (e.g. in this func name compute can become comp and optimal can become opt)